City-of-Pori/pori-events
# PORISD-77: Node packages mass update

## Test
1. Go to `drupal/web/themes/custom/pori_events`
2. Run `lando gulp` or `lando gulp --production`to see that watcher works (no errors).
3. Change any `scss` file to see that watcher still works.

# Story
https://wunder.atlassian.net/browse/PORISD-77 - Pori events - Node packages mass update

Story related w dependabot alerts in https://github.com/City-of-Pori/pori-events/security/dependabot:
![image](https://user-images.githubusercontent.com/22578013/112839973-da3c6600-90af-11eb-9b9e-b48a13aac32e.png)
